App.tsx

```diff
@@ -4,9 +4,9 @@ import React from 'react';
 import { DefaultTheme, Portal, Provider as PaperProvider } from 'react-native-paper';
 import NetInfo from '@react-native-community/netinfo';
 import _ from 'lodash';
-import { storedGutka, entryObj, gutkaEntry, SearchType, QueryType } from './config/types';
-import { fetchGutkas, saveGutkas, fetchSettings, findCurrentGutka, getGutkaItems, findCurrentGutkaIndex, findEntry } from './functions';
-import { downloadDB, checkIfDbExists, loadShabad, downloadProg, query } from './config/database/database';
+import { storedGutka, entryObj, gutkaEntry, SearchType, QueryType } from './config/dev_env/types';
+import { fetchGutkas, saveGutkas, fetchSettings, findCurrentGutka, getGutkaItems, findCurrentGutkaIndex, findEntry } from './config/app_state/functions';
+import { downloadDB, checkIfDbExists, loadShabad, downloadProg, query, testSchemas } from './config/database/database';
```

Comment:
need to really clean up these imports (remove the now unused etc and use the new defined conventions in #28